apps/token-manager/app/src/components/SideBar.js

```diff
@@ -16,12 +16,11 @@ const DISTRIBUTION_COLORS = [
 ]
 
 const displayedStakes = (accounts, total) => {
-  const positiveAccounts = accounts.filter(({ balance }) => balance > 0)
```

Comment:
I think this should've actually been `balance.gtn(0)` anyway, but this is actually already [handled in the script](https://github.com/aragon/aragon-apps/blob/master/apps/token-manager/app/src/script.js#L102).